thirdeye/thirdeye-frontend/app/pods/partials/rootcause/dimensions/template.hbs

```diff
@@ -3,6 +3,12 @@
     {{#subtablist.tab name="heatmap"}}Heatmap{{/subtablist.tab}}
     {{#subtablist.tab name="table"}}Table{{/subtablist.tab}}
     {{#subtablist.tab name="algorithm"}}Algorithm{{/subtablist.tab}}
+    <span>
+      <i class="glyphicon glyphicon-question-sign"></i>
+      {{#bs-popover placement="bottom" title="What do the options mean?"}}
+        {{cube-tooltip}}
+      {{/bs-popover}}
+    </span>
```

Comment:
Is it possible to put this question sign next to
1. "Color heatmap by" when the user selects the heatmap tab and
2. At the end of the cell "Contribution to Overall Change" and "Change in Contribution?" when in the Algorithm tab.
Thanks.